flatten.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+if [ ! -d "flats" ]; then
```

Comment:
Does it make sense to extend your change to remove the directory content only if the directory exists:

```
if [ ! -d "flats" ]; then
    mkdir flats
else
    rm -rf flats/*
fi
```

Reply:
It is what this script does : 
If the folder doesn't exist,the script doesn't fail and creates the directory. That is my change.

If the folder already exists, removes its content and flatten the .sol.

It is useful in case you have a newer version of the SC but the directory already exists.